packages/account/src/hooks/useKycAuthStatus.ts

```diff
@@ -7,15 +8,21 @@ type TKycAuthStatusPayload = TSocketRequestPayload<'kyc_auth_status'>['payload']
 /** Custom hook that returns Proof of Identity (POI) and Proof of Address (POA) authentication status details. */
 export const useKycAuthStatus = (payload?: TKycAuthStatusPayload) => {
     const { client } = useStore();
+    const invalidate = useInvalidateQuery();
 
     const { is_authorize } = client;
     const { data, ...kyc_auth_status_rest } = useQuery('kyc_auth_status', {
         payload,
         options: { enabled: is_authorize },
```

Comment:
May I please ask you just to double check that there is no issues when you reload the browser's page being on proof-of-address page 🙏 
just to make sure, that we do not make 'kyc_auth_status' call before 'authotize'

Reply:
It gets called after authorization